operators/src/processing/raster_vector_join/mod.rs

```diff
@@ -86,14 +88,40 @@ impl VectorOperator for RasterVectorJoin {
         // TODO: check for column clashes earlier with the result descriptor
         // TODO: update result descriptor with new column(s)
 
+        let raster_sources = self
+            .raster_sources
+            .drain(..)
+            .map(|source| source.initialize(context))
+            .collect::<Result<Vec<_>>>()?;
+
+        let result_descriptor = vector_source.result_descriptor().map_columns(|columns| {
+            let mut columns = columns.clone();
+            for (i, new_column_name) in self.params.names.iter().enumerate() {
+                let feature_data_type = match self.params.aggregation {
+                    AggregationMethod::First => {
+                        match raster_sources[i].result_descriptor().data_type {
+                            RasterDataType::U8
+                            | RasterDataType::U16
+                            | RasterDataType::U32
+                            | RasterDataType::U64
+                            | RasterDataType::I8
+                            | RasterDataType::I16
+                            | RasterDataType::I32
+                            | RasterDataType::I64 => FeatureDataType::Decimal,
```

Comment:
remark: I still find it confusing that our integer types are named Decimal when e.g. in Postgres Decimal is a variable precision data type with a decimal point.

Reply:
True, maybe we can sit together and think of reworks to the feature data types for a small story.